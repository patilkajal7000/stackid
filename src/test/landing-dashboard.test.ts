import { assert, describe, expect, it } from 'vitest';

describe('Landing Dashboard', () => {
    it('foo', () => {
        assert.equal(Math.sqrt(4), 2);
    });

    it('bar', () => {
        expect(1 + 1).eq(2);
    });

    it('snapshot', () => {
        expect({ foo: 'bar' }).toMatchSnapshot();
    });
});
