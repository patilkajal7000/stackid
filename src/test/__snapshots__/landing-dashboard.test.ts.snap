// Vitest Snapshot v1

exports[`Landing Dashboard > snapshot 1`] = `
{
  "foo": "bar",
}
`;
